src/components/ExecutePassButton.js

```diff
@@ -1,18 +1,12 @@
 import React from 'react';
 import Card from './Card';
+import { useGameContext } from '../context/GameContext.js';
 
-export default function ExecutePassButton({
-  passCard,
-  setFrom,
-  from,
-  to,
-  selectedCard,
-  setSelectedCard,
-}) {
+export default function ExecutePassButton({ passCard }) {
+  const { selectedCard, setSelectedCard, from, to } = useGameContext();
   return (
     <div className="execute-button" onClick={() => passCard(selectedCard)}>
-      Pass <Card card={selectedCard} setSelectedCard={setSelectedCard} setFrom={setFrom} /> from{' '}
-      {from} to {to}
+      Pass <Card card={selectedCard} setSelectedCard={setSelectedCard} /> from {from} to {to}
```

Comment:
You don't need setSelectedCard because you are getting it from context in the Card component